Replace sound effects

jgengelhardt opened the pull request:
Replaces sound effects from the Tux table with new sound effects created in Audacity from field recordings to resolve #8. Sources and licenses are included in /data/hurd/README.txt

jgengelhardt committed:
Update sounds, textures, and update licensing

revert behavior in trunk

typo

rzr commented:
Great I will test it out

Btw may I suggets to use reuse tool

https://reuse.software/

I will generate .licence files next to binary files

jgengelhardt commented:
Good call. I've started working on REUSE compliance.

rzr commented:
Feel free to ask any help

jgengelhardt commented:
I've added license information to the new and changed data/hurd/ files and made obvious corrections to the formatting of some existing copyright and licensing headers. However, I'm not confident enough to add information to the rest of the files in the repo. Please let me know if any additional corrections are needed.

rzr commented on data/hurd/gnu.png.license:
```diff
@@ -1,5 +1,6 @@
 Copyright 2003 Free Software Foundation, Inc,
 Copyright 2003 Aurelio A. Heckert <aurium@gmail.com>,
 Copyright 2010 Paolo Caroni <kenren89@gmail.com>
+Copyright 2022 Joseph Engelhardt
```
please add email

rzr replied:
did you change those files ? if not i am not sure your copyright  should be listed here

jgengelhardt replied:
I did change gnu.* and hurd.* to combine elements of the gray and red tables.

rzr replied:
ok so it's a derivate work since you're not author of the main I think only the only creator should be preserved, what do you think ?

jgengelhardt replied:
I'm okay with that if you are.

rzr replied:
ok thx can you please squash commits to make it clearer and I'll test it this weekend

jgengelhardt commented:
This fork also combines textures from the red and gray versions of the table, to resolve #13 and increase contrast and readability of the table. The new color scheme combines elements of Paolo's separate red and gray designs, and before and after screenshots are available in the comments for issue #13.

jgengelhardt commented:
Just squashed commits into one for neatness. This includes the new sound and music, gray textures from a previous version, and updated licensing.

rzr commented:
Hi, I've been testing your change. sorry if i was long I see you also modified the cpp file can you please update commit message too telling why you wanted a different behaviour ?

rzr commented:
ok thx

rzr commented:
Sorry I missed those changes for current release... I wanted minimal changes with SDL2 

your changes are still good, I'll merge them give me a litle time

rzr commented:
Sorry it took so long :) I'll  might probably wait SDL3 to release a new version...
